contract/src/test/java/io/neow3j/contract/ContractManagementTest.java

```diff
@@ -14,6 +14,7 @@
 import io.neow3j.types.ContractParameter;
 import io.neow3j.types.Hash160;
 import io.neow3j.wallet.Account;
+import org.junit.jupiter.api.Assertions;
```

Comment:
chore: please use the static import `org.junit.jupiter.api.Assertions.assertFalse` instead.